### DISCUSSION TITLE
Briefly seeing Failed to process BRAW File

### DISCUSSION THREAD

**Discussion by tomwolsky** [latenitefilms/BRAWToolbox#36] [Support] [answered]
When I select a 6K clip in list view the filmstrip may shown a Failed to process BRAW File image, which will also be in the Viewer. As soon as I scrub the filmstrip it will disappeared. Occasionally the image will pop up while I'm playing or scrubbing a project. I'm on an M1 Max MBP.
<img width="241" alt="60%_Screenshot 2023-01-25 at 15 46 48" src="https://user-images.githubusercontent.com/18217685/214688864-c27b8905-c686-478a-8a30-fa5f82f7fb35.png">

**Comment by latenitefilms** [comment, marked as answer]
Try this: https://brawtoolbox.io/troubleshooting/#bad-frames-in-the-final-cut-pro-browser

**Reply by Marks07-stack** [reply]
M1 MBP on Ventura 13.1 FCP 10.6.5 - Tried importing some 5.7 K BrW  and getting the red error frames for all clips. I have tried the thumbnail deletion suggestion to no avail. I've got audio in my clips but no picture.  Any more suggestions to fix the problem? The error indicated is " no Braw file selected."

**Reply by latenitefilms** [reply]
If you have a "No BRAW File Selected", I'm GUESSING that you have just applied the BRAW Toolbox effect to a clip instead of using the Workflow Extension?

Follow the instructions here: https://brawtoolbox.io/installation/

**Reply by tangierc** [reply]
Hi Chris. I keep getting the "No BRAW File Selected" problem with clips in some of my events. I haven't used the effect at all. Everything as been imported using the Workflow Extension.  When I attempt to relink clips with an event and grant access to the drive or folder containing the clips, it doesn't fix the issue.

**Comment by Marks07-stack** [comment]
second question:  Workflow extension is working- the second question is this: I have a doc project shot in Braw. The director and I have copies of the Braw camera originals . If I set up a library in X with the camera originals using Braw tool box, how would I share the library/ projects with him? Ultimately we would like to send xml's back and forth for versions of cuts .

**Reply by latenitefilms** [reply]
As long as you both have files on same path (ie an external drive with same name) it should work well.

See: https://brawtoolbox.io/collaboration/
